js/index.js

```diff
@@ -1,26 +1,40 @@
-const today = new Date();
-const thisYear = today.getFullYear();
-const footer = document.querySelector('footer');
-const copyRight = document.createElement('p');
+// copyright on footer
+const renderCopyright = () => {
+    const today = new Date();
+    const thisYear = today.getFullYear();
+
+    const copyright = document.querySelector('#copyright')
+    
+    copyright.innerHTML = `&copy; Raissa Gomes ${thisYear}`
+
+}
+
+renderCopyright();
+
+//skills section
+const renderSkillsList = () => {
+const skills = ['JavaScript', 'HTML', 'CSS', 'GIT']
 
-copyRight.innerHTML = `&copy Raissa Gomes ${thisYear}`;
-footer.appendChild(copyRight);
 
-const skills = ['HTML' 'CSS' 'JavaScript'];
+//DOM
 const skillsSection = document.getElementById('skills');
 const skillsList = skillsSection.querySelector('ul')
 
 for (let i = 0; i < skills.length; i++) {
     const skill = document.createElement('li');
     skill.innerHTML = skills[i];
     skillsList.appendChild(skill);
+
+ }
 }
 
+renderSkillsList();
+
 const messageForm = document.getElementsByName('leave_message');
```

Comment:
You changed your form name in the HTML to `"leave-a-message"` so you'll need to change this code in the Javascript as well otherwise it won't find any element on line 33. Either that or change the HTML to match what's expected by this JS line